ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/db/impl.clj

```diff
@@ -0,0 +1,26 @@
+(ns com.sixsq.slipstream.ssclj.db.impl
+  (:require
+    [com.sixsq.slipstream.ssclj.db.binding :as p]
+    [clojure.tools.logging :as log]))
+
+(def ^:dynamic *impl*)
+
+(defn set-impl!
+  [impl]
+  (alter-var-root #'*impl* (constantly impl)))
```

Comment:
The variable probably could be bound from the top-level ring application, but this is only changed once when the application starts based on values read in from the service configuration.